dev/initdata/import-thirdparties.php

```diff
@@ -149,7 +149,7 @@
         $condpayment = trim($fields[36]);
         if ($condpayment == 'A la commande') $condpayment = 'A réception de commande';
         if ($condpayment == 'A reception facture') $condpayment = 'Réception de facture';
-        $object->cond_reglement_id = dol_getIdFromCode($db, $condpayment, 'c_payment_term', 'libelle_facture', 'rowid');
+        $object->cond_reglement_id = dol_getIdFromCode($db, $condpayment, 'c_payment_term', 'libelle_facture', 'rowid', getEntity('c_payment_term'));
```

Comment:
Why adding here a parameter "getEntity('c_payment_term')"
The key to identify the dictionary is already provided in parameter 3 ('c_payment_term').
We don't need to add a parameter, just reuse the parameter 3.

Reply:
@eldy 
I can not use "getEntity" in the function because it would force me to make a permanent query on the "entity" field and not all of the potential tables have the "entity" field! you see ?

Reply:
I see.
However having the logic to find the entity to filter a table outside the call of the function that read the table is hurting me.
Can't we just add a parameter to say "restrict to qualified entities" that will be 1 and move the call to getEntity into function  dol_getIdFromCode ?

$object->cond_reglement_id = dol_getIdFromCode($db, $condpayment, 'c_payment_term', 'libelle_facture', 'rowid', '1');

Reply:
@eldy 
ok it's ready